lib/protocol/routes.js

```diff
@@ -439,7 +439,7 @@ const METHOD_MAP = {
     }
   },
   '/wd/hub/session/:sessionId/appium/device/app_state': {
-    GET: {command: 'queryAppState',
+    POST: {command: 'queryAppState',
```

Comment:
Wouldn't this be a breaking change for all of the other clients?

I would rather do both:

```javascript
'/wd/hub/session/:sessionId/appium/device/app_state': {
  GET: {command: 'queryAppState', ...},
  POST: {command: 'queryAppState', ...},
```

Reply:
it will be, but the endpoint has been added just recently, so it is not going to do any harm. Do you think we should keep the GET one?

Reply:
If no other client uses it then I'm fine with just tossing the `GET` line.

Reply:
Looks like the python client uses it:

https://github.com/appium/python-client/blob/f8288d5ab975c8ccfad83d9fd7d631ef987175ea/appium/webdriver/webdriver.py